include/sys/fs/zfs.h

```diff
@@ -192,6 +192,11 @@ typedef enum {
 	ZFS_PROP_REDACT_SNAPS,
 	ZFS_PROP_SNAPSHOTS_CHANGED,
 	ZFS_PROP_PREFETCH,
+	ZFS_PROP_BROWSE,		/* macOS: nobrowse/browse */
+	ZFS_PROP_IGNOREOWNER,	/* macOS: ignoreowner mount */
+	ZFS_PROP_LASTUNMOUNT,	/* macOS: Spotlight required */
+	ZFS_PROP_MIMIC,			/* macOS: mimic=hfs|apfs */
+	ZFS_PROP_DEVDISK,		/* macOS: create IOkit virtual disk */
```

Comment:
Totally guessing from the comment that this is about making a zvol visible to the rest of the system. If that's so, would it be better done as a new value for `volmode`?

Reply:
Same as above. Needed for backwards compatability with current zfs on mac pools.

Reply:
As it turns out its not actually a variant of `volmode`, but some mount thing. So probably fine.